website/docs/Threshold Decryption.md

```diff
@@ -0,0 +1,75 @@
+---
+slug: Threshold Decryption.md
+sidebar_position: 16
+---
+
+Threshold Decryption (TDec) is a programmable access control service, in which decryption rights are typically based on the verified fulfillment of predefined conditions. Access conditions can be EVM-based (e.g. does the designated recipient own this NFT), RPC-driven (e.g. did the designated recipient commence active mining before this block) or time-based (e.g. has a preset period of time elapsed, after which the recipient's requests will be ignored). These conditions are also composable and can be combined in any logical sequence or decision tree.  
+
+TDec involves splitting the secret (a decryption key) into multiples shares and distributing those among authorized and collateralized node operators (stakers in the Threshold network). A minimum number – a threshold – of those operators holding the key shares must be online and actively participate in order to reconstruct the original secret.
```

Comment:
Maybe not "reconstruct the original secret", but rather each perform partial decryptions that can be combined by the requester to reconstruct the original plaintext data.

The original secret (a decryption key) is never reconstituted.